ec/internal/validators/knownvalidator.go

```diff
@@ -53,6 +50,6 @@ func (v knownValidator) Validate(ctx context.Context, req tfsdk.ValidateAttribut
 //   - Is known.
 //
 // Null (unconfigured) values are skipped.
-func Known() tfsdk.AttributeValidator {
+func Known() validator.String {
```

Comment:
Similarly should return a concrete type and include a type assertion at the top of this file.